data/woelfel2020.yaml

```diff
@@ -18,19 +18,28 @@ analytes:
     limit_of_quantification: 100
     limit_of_detection: unknown
     specimen: stool
+    biomarker: SARS-CoV-2
+    unit: gc/mL
+    reference_event: symptom onset
   sputum:
```

Comment:
Is this going to be changed to the target name? Like N1-XXXXXX (for N gene)? As this information was in the "specimen" field?

Reply:
Yes, good question. How specific do you think we want to be here? Having just `SARS-CoV-2` could be nicer for filtering at a later point, but `SARS-CoV-2-N1-xxx` is more specific. What do you think?

Reply:
I think SARS-CoV-2-N1 works. Do you want to also add "target: N1" along with the biomarker?

Reply:
Sounds good!

Reply:
That's updated. The authors mention quantification targeting the E and RdRP genes, but I couldn't find an explicit reference to which one was being reported in the figure. Have you seen a reference?

Reply:
I also can not find the reference. Actually, I am a little confused about how they quantify the concentration with two genes.

Reply:
Sounds good!

Reply:
Thanks for the updates to the schema! I'll be using this updated schema to format the Arts dataset accordingly.